libs/design/src/molecules/select/selected-option/selected-option.directive.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  Directive,
+  HostBinding,
+  Optional,
+  TemplateRef,
+} from '@angular/core';
+
+/**
+ * A directive for styling the content of the selected option.
+ */
+@Directive({
+  selector: '[daffSelectSelectedOption]',
```

Comment:
Doesn't seem like this component actually works? I'm looking at the examples and the select option doesn't have the directive selector or the class anywhere on the selected option.